scripts/sync-postgres-exporter-mixin.sh

```diff
@@ -0,0 +1,57 @@
+#! /usr/bin/env bash
+
+set -euo pipefail
+
+exporter="tmp/postgres_exporter"
+mixin="${exporter}/postgres_mixin"
+
+cd "$(git rev-parse --show-toplevel)"
+
+if [ -d "$exporter" ]; then
+	rm -fr "$exporter"
+fi
+mkdir -p tmp
+git clone --depth 1 https://github.com/prometheus-community/postgres_exporter.git tmp/postgres_exporter
+
+# This mixin requires a few tools to be installed, we need to check for those
+# and print out that they need to be installed if they are not.
+
+if [ ! -x "$(command -v go)" ]; then
+	echo "go is not installed, please install it"
+	exit 0
+fi
+
+if [ ! -x "$(command -v mixtool)" ]; then
+	echo "mixtool is not installed, please install it"
+	echo "https://github.com/prometheus-community/postgres_exporter/tree/master/postgres_mixin"
+	exit 0
+fi
+
+# TODO(nhudson) think about adding in jq formatting to make the dashboard
+# code look nicer.
+#if [ ! -x "$(command -v jq)" ]; then
+#	echo "jq is not installed, please install it"
+#	exit 0
+#fi
+
+cd "$mixin"
+make build
+cd -
+
+# This seems like the most straightforward way to replace the datasource in the
+# generated dashboard for now. We can revisit this if we find a better way to
+# replace the generated datasource.
+for file in `ls ${mixin}/dashboards_out/*.json`
+do
+	cp -r $file chart/dashboards/$(basename $file)
+	sed -i.orig 's/\"datasource\": .*$/\"datasource\": {\n    \"type\": \"prometheus\",\n    \"uid\": \"dc08d25c8f267b054f12002f334e6d3d32a853e4\"\n }, /g' chart/dashboards/$(basename $file)
+	find chart/dashboards/ -name '*.orig' -exec rm -f {} \;
+done
```

Comment:
We do have this logic in https://github.com/timescale/tobs/blob/main/scripts/sync-dashboards.sh, maybe we can extract it to some `make` target or some other script and align both. WDYT?

Reply:
Yeah syncing both scripts into one and adding a make target sounds like a good idea.  I will get that knocked out.